src/splits.rs

```diff
@@ -430,20 +434,21 @@ pub enum Split {
     /// Rescued Flea Underworks Explosions (Flea)
     ///
     /// Splits after rescuing flea in Under_21
-    SavedFleaUnderworksExplosions,
+    #[alias = "SavedFleaUnderworksExplosions"]
+    SavedFleaUnderworksCauldron,
     /// Rescued Flea Underworks Wisp Thicket (Flea)
     ///
     /// Splits after rescuing flea in Under_23
     SavedFleaUnderworksWispThicket,
-    /// Defeated Giant Flea (Flea)
+    /// Rescued Giant Flea (Flea)
     ///
     /// Splits after defeating Giant Flea
     SavedFleaGiantFlea,
```

Comment:
I'm not sure what the deal is with this giant flea, but there seems to be a mismatch between the split name `Rescued` and the split description `defeating`. If you change one, should you change the other?